README.md

```diff
@@ -35,6 +71,8 @@ which results in truncated labels, so make sure to embed fonts.
 For drawio a 
 [desktop installation](https://github.com/jgraph/drawio-desktop) is also available.
 
-The latest editor's draft is available [here](https://w3c.github.io/wot-discovery/).
+## Known Implementations
 
-The latest Implementation Report is available [here](https://w3c.github.io/wot-discovery/testing/report.html).
+The W3C WoT collects known implementations at <https://www.w3.org/WoT/developers/>. Implementations of Discovery are found under categories 
+"TD Directory (TDD) Implementations", "Runtime Implementations for TD Consumers" and "Runtime Implementations for TD Exposers". Further information
+can also be found in the [implementations](implementations/README.md) folder. 
```

Comment:
These are pretty outdated. I suggest removing this file and maintaining the implementations in one place only.